lib/wallet.js

```diff
@@ -1,4 +1,7 @@
 module.exports = function(options) {
+  var EC = require('elliptic').ec;
+  var ec = new EC('secp256k1');
```

Comment:
This looks like it's uncompressing the compressed pub key, only to compress it again for hashing ?